src/main/java/hw_5/myCity/Building.java

```diff
@@ -0,0 +1,21 @@
+package hw_5.myCity;
+
+import hw_5.myClass.Person;
+
+public class Building {
+    Person[] residents;
+
+    public Building(int numberOfResidents) {
+        residents = new Person[numberOfResidents];
+    }
```

Comment:
You don't need these elements. 

![Screenshot_5](https://github.com/user-attachments/assets/9b4f7b69-d250-4eda-9995-9c988819952a)

just create a main method inside Building and inside main method create Persons and add persons to the array. Using this array display persons

Reply:
done